reporting/reports.py

```diff
@@ -3,6 +3,7 @@
 import re
 import sys
 
+from six import reraise as raise_
```

Comment:
Why is this alias? I'd rather prefer to use it `six.reraise` instead.

Reply:
Using `raise_with_tb` implementation alternative from `spacewalk-usix` to keep consisntecy.

Reply:
I am still against this. We should start using `six`. Either update `usix` package with a proper `six` or make sure `python-six` is available. In any case, this should be a proper migration.